### PR TITLE
Am 2763 sscs wa mapping rules judicial

### PR DESCRIPTION
**Pull request by abul-syed**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/AM-2763

### Change description ###

SSCS_WA - Add mapping rules in ORM for SSCS judges

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by adityadwadasi — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialRoleMappingTest.java [hunk 1]
```diff
@@ -87,11 +146,45 @@ void shouldReturnFeePaidRoles(String setOffice, String roleNameOutput) {
...
+            assertEquals("Fee-Paid", r.getAttributes().get("contractType").asText());
+            assertCommonRoleAssignmentAttributes(r, regionId, setOffice);
+        });
+    }
 
+    @ParameterizedTest
+    @CsvSource({
+            "SSCS Tribunal Judge-Fee Paid,'fee-paid-judge,judge,hmcts-judiciary'",
+    })
+    void shouldReturnFeePaidRolesWithBooking(String setOffice, String expectedRoles) throws IOException {
```
Comment (tomxelliott) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialRoleMappingTest.java:164]: Could this test be combined with the one above? a conditional to check the bits for with booking vs without? If we anticipate more scenarios being added to the shouldReturnFeePaidRolesWithBooking test in future then I am happy to leave it as is

**Review by tomxelliott — APPROVED** [review]

**Review by mattnayler — APPROVED** [review]
Additional review comments can be actioned under patch to SSCS_WA_1_0 feture branch: see PR #1496.

Also legacy issues:
* if `sscs_judge_joh` rule still required should it have usual endTime filter.
* if `sscs_multi_region_dependency_org_role_wales` and `sscs_multi_region_dependency_org_role_southwest` still required can we ensure there is a unt test.

File: src/main/resources/validationrules/sscs/sscs-judicial-office-holder-mapping.drl [hunk 2]
```diff
@@ -267,6 +267,8 @@ when
    $f:  FeatureFlag(status && flagName == FeatureFlagEnum.SSCS_WA_1_0.getValue())
    $jap: JudicialAccessProfile(appointment == "Tribunal Member Financially Qualified",
                                appointmentType == "Fee Paid",
+                               baseLocationId == "1032",
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/main/resources/validationrules/sscs/sscs-judicial-office-holder-mapping.drl:270]: This seems to be an extra rule from the old config. Which has been removed by the BA in the updated config, hence have removed. See: [e0a3add](https://github.com/hmcts/am-org-role-mapping-service/pull/1493/commits/e0a3add1c3f95db1f393730b0357af1be7357c70)

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java [hunk 3]
```diff
@@ -25,53 +25,6 @@
 @RunWith(MockitoJUnitRunner.class)
 class DroolSscsJudicialOfficeMappingTest extends DroolBase {
 
-    //=================================SALARIED ROLES==================================
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java:28]: added back in [74c13e5](https://github.com/hmcts/am-org-role-mapping-service/pull/1493/commits/74c13e5a5bb444258a1b9e740e0776ba34eec759). only removed work types check, as there is already a thorough work type check in `DroolSscsJudicialRoleMappingTest`

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java [hunk 4]
```diff
@@ -112,60 +63,8 @@ void shouldReturnTribunalMemberMedicalRolesNoAccessRequests() {
             }
         });
 
-        assertEquals("hearing_work,decision_making_work,routine_work,priority",
-                roleAssignments.get(3).getAttributes().get("workTypes").asText());
-    }
-
-    //=================================FEE-PAID ROLES==================================
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java:119]: added back in [74c13e5](https://github.com/hmcts/am-org-role-mapping-service/pull/1493/commits/74c13e5a5bb444258a1b9e740e0776ba34eec759). only removed work types check, as there is already a thorough work type check in `DroolSscsJudicialRoleMappingTest`

File: src/main/resources/validationrules/sscs/sscs-judicial-office-holder-mapping.drl (hunk 2 above)
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/main/resources/validationrules/sscs/sscs-judicial-office-holder-mapping.drl:270]: I have left `sscs_judge_joh ` untouched. as discussed - its behind the `SSCS_HEARING_1_0` flag. maybe something to look into, part of tech debt

[the remark this answers is not in the document]
Comment (mattnayler) [reply src/main/resources/validationrules/sscs/sscs-judicial-office-holder-mapping.drl:270]: Comment transfered to #1496.

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java:28]: Comment transfered to #1496.

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java:119]: Comment transfered to #1496.

**Pull request merged**